source/Generic/PlayState/PlayState.csproj

```diff
@@ -93,6 +91,17 @@
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </None>
   </ItemGroup>
+  <ItemGroup>
+    <PackageReference Include="Microsoft.Toolkit.Uwp.Notifications">
+      <Version>7.1.2</Version>
+    </PackageReference>
+    <PackageReference Include="PlayniteSDK">
+      <Version>6.2.2</Version>
+    </PackageReference>
+    <PackageReference Include="System.Management">
+      <Version>5.0.0</Version>
+    </PackageReference>
```

Comment:
Is this reference used in the project?

Reply:
It was added automatically when adding the dependencies for Windows notifications, so I assume yes. Anyway I will make a test without it just to make sure.